examples/named_entity_recognition/Named_Entity_Recognition_with_Tensorflow_and_🔭_Galileo.ipynb

```diff
@@ -23,6 +23,14 @@
       "outputs": [],
```

Comment:
<p>Seems like we don't even use <code>data["validation"]</code> actually in this one. We create <code>tf_validation_set</code> but dont do anything with it! We can just delete it and only keep <code>train</code> and <code>test</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rungalileo/examples/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>